api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentGroupControllerTest.java

```diff
@@ -622,6 +622,331 @@ public void addNewPaymenttoExistingPaymentGroupTest() throws Exception {
         assertEquals("Amount saved in remissionDbBackdoor is equal to the on inside the request", amount, paymentsResponse.getAmount());
     }
 
+    @Test
+    public void addNewPaymenttoExistingPaymentGroupWhenServiceTypeIsFinrem() throws Exception {
+        PaymentGroupDto request = PaymentGroupDto.paymentGroupDtoWith()
```

Comment:
I think we are testing the logic to 'addNewPaymenttoExistingPaymentGroup' for different Services, can we just extract the common logic in a method, this will add some value to Code readability.

Reply:
@AyanDuttaHMCTS - Fixed.